shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -76,6 +79,7 @@ private HapiHelper() {}
      */
     public static <T extends Resource> Stream<T> resourcesInBundle(
             Bundle bundle, Class<T> resourceType) {
+        // null check?
```

Comment:
I added this comment here so I could get opinions on this. I feel we should have a null check here and return an empty stream if null, thoughts? The reason for this is that when I was trying to mock resources, I kept getting null pointer exception here. Eventually, I decided not to use the mocking framework for the unit tests but the question still remained.

Reply:
I ended up adding the null check.